app/grandchallenge/algorithms/templates/algorithms/partials/algorithminterface_table.html

```diff
@@ -12,53 +12,44 @@
         </thead>
         <tbody>
             {% for interface in interfaces %}
-                 <tr>
-                    <td class="text-center align-middle p-3">{{ forloop.counter }}</td>
-                    <td>
+                <tr>
+                    <td class="text-center p-1">{{ forloop.counter }}</td>
+                    <td  class="px-1">

```

Comment:
Has this file had a linter run on it? The indentations have slightly changed and there are extra spaces here. Surprised our CI didn't pick up on it.

Reply:
I ran pre-commit on it, as usual. Strange that didn't detect these things.

Reply:
Still some spaces removed, looking into it

Reply:
Hm I actually think the current indents are correct. 4 spaces. Could it be that the original file was wrong?

Reply:
Could be!

Reply:
Related: #3914